src/uu/mv/Cargo.toml

```diff
@@ -25,6 +25,12 @@ uucore = { workspace = true, features = [
   "update-control",
 ] }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+winapi = { version="0.3" }
```

Comment:
can we do it with winapi_util?
I am a bit reluctant to add a new dep

Reply:
`winapi_util` seems like no `ERROR_NOT_SAME_DEVICE`. maybe i can use `uucore::windows_sys::Win32::Foundation::ERROR_NOT_SAME_DEVICE`?